src/App.css

```diff
@@ -1,6 +1,65 @@
-.auth-container{
-    background-image: url('./assets/images/bg3.png');
-    background-position:0ex;
-    height: 100vh;
+/* ---------------------- common style ---------------------- */
 
+.main-colr {
+    color: rgba(239, 155, 40, 1);
```

Comment:
what about making it a css variable instead ?

Reply:
The color has been successfully converted into a CSS variable